README.md

```diff
@@ -78,6 +78,7 @@ COMMUTER_BUCKET=sweet-notebooks commuter
 | `COMMUTER_S3_KEY`            | AWS Key                                         | Optional, uses IAM roles or `~/.aws/credentials` otherwise |
 | `COMMUTER_S3_SECRET`         | AWS Secret                                      | Optional, uses IAM roles or `~/.aws/credentials` otherwise |
 | `COMMUTER_S3_ENDPOINT`       | S3 endpoint                                     | Optional, selected automatically                           |
+| `COMMUTER_S3_FORCE_PATH_STYLE`| Set to `true` to activate `s3ForcePathStyle`     | `false`                                                      |
```

Comment:
Nit: Can you make the description for`s3ForcePathStyle` explain what it does? E.g. "forces path-style URLs for S3 objects" from the AWS docs.

Reply:
Thanks for the comment. I improved the description, hope it's more clear now.